code/src/constants/orderStatuses.ts

```diff
@@ -1,7 +1,7 @@
 export const orderStatuses = {
-  IN_PROGRESS: "in progress",
-  SHIPPED: "shipped",
-  DELIVERED: "delivered",
-  CANCELED: "canceled",
-  COMPLETED: "completed"
+  IN_PROGRESS: "In progress",
+  SHIPPED: "Shipped",
+  DELIVERED: "Delivered",
+  CANCELED: "Canceled",
+  COMPLETED: "Completed"
```

Comment:
should be translations for statuses

Reply:
We are using shortcut for background. Look for example AppButton or other app components

Reply:
We are using shortcut for background. Look for example AppButton or other app components

Reply:
added